.claude/commands/cleanup.md

```
@@ -0,0 +1,75 @@
+2. **WSL scratch clones** (see the `wsl-docker-linux-verification` project memory): a clone
+   made under `~/verify` or similar for one-off CI-matching verification has no reason to
+   persist once the check is done. Remove it the same session, not "next time someone
+   notices."
+   **Root-owned files from a Docker bind mount**: if the clone was built inside a
+   `docker run` container (this repo's Linux-verification recipe does exactly this), the
+   container's writes (target/, build artifacts) are owned by root inside the container — a
+   plain `rm -rf` from the WSL user account fails with `Permission denied` on every such file.
+   **Do not reach for `sudo`** (see "Commands needing a password/interactive auth" in
+   `.claude/rules/index.md`) — Docker itself already runs as root, so a second throwaway
+   container can delete what the first one wrote, with no host password needed:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)